### DISCUSSION TITLE
Command to set the Autonukemode default spell

### DISCUSSION THREAD

**Discussion by Shockii** [cyritegamestudios/trust#103] [Ideas]
Opening the console and cycling through the default elements takes a bit of time, is there a way to add in a cmd to let us set the spell type using a console cmd using a macro, i.e /console trust autonukemode Wind

**Comment by cyrite** [comment]
Yup! You can do // trust set AutoNukeMode Earth. This works for any mode.

**Reply by Shockii** [reply]
Ahh I see how to set the commands now, many thanks. 

I find that without Autonukermode on, I basically just auto attack and never use any of my spells though. Though with autonukermode on I just spam spells. Is there a way to set a delay on autonukermode or a means of getting spells to cast more frequently but not as often as spamming them?

**Reply by cyrite** [reply]
You could change the default`Delay` in `NukeSettings`. For example, for **Black Mage** you would do that in the `BLM.lua` [settings file](https://github.com/cyritegamestudios/trust/blob/8fc0dd15d04a4532dac4214c4ba7dfdbc31b1bfe/data/BLM.lua#L19). Another option is to make a skillchain and [magic burst ](https://github.com/cyritegamestudios/trust/wiki/AutoMagicBurstMode) instead.

**Reply by 21gh0st21** [reply]
Would that command work for JA spells?
